test/com/core/tictactoe/BoardTests.java

```diff
@@ -0,0 +1,171 @@
+package com.core.tictactoe;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.hamcrest.CoreMatchers.isA;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+
+public class BoardTests {
+
+    private Board board;
+    private Player player;
+    private Player player2;
+
+    @BeforeEach
+    void instantiateBoard() {
+        board = new Board();
+        player = new Player("X");
+        player2 = new Player("Y");
+    }
+
+    @Test
+    void BoardIsCreatedWithPlacesArray() {
+        assertThat(board.places,isA(ArrayList.class));
+    }
+
+    @Test
+    void BoardHasFilledPlacesWhenCreated() {
+        for(int i = 0; i < 9; i++) {
+            assertEquals(board.places.get(i),i);
+        }
+    }
+
+    @Test
+    void changesSignOnBoardToPlayerSign() {
+        board.putSignOnBoard(player, 0);
+
+        assertEquals("X", board.places.get(0));
+    }
+
+
+
+    @Test
+    void checksIfThereIsATieFalse() {
+        board.putSignOnBoard(player, 0);
+
+        board.isTie();
+
+        assertFalse(board.isTie());
+    }
+
+    @Test
+    void checksIfThereIsATieTrue() {
+        int[] array1 = {1, 3, 4, 6, 8};
+        int[] array2 = {0, 2, 5, 7};
+        playWholeGame(player, player2, array1, array2);
+
+        board.isTie();
+
+        assertTrue(board.isTie());
+    }
+
+
+    @Test
+    void checkIfGameIsWonFalse() {
+        playMoves(player, 0, 5, 2);
+
+        assertFalse(board.isWon());
+    }
+
+    @Test
+    void checkIfGameIsWonTrueV1() {
```

Comment:
V1, V2, etc is not very descriptive.  Could you improve on these?

Reply:
done.